test/bridges/base-bridge-receiver-test.ts

```diff
@@ -0,0 +1,452 @@
+  const timelock = await TimelockFactory.deploy(
+    admin,              // admin
+    10 * 60,            // delay
+    14 * 24 * 60 * 60,  // gracePeriod
+    10 * 60,            // min delay
+    30 * 24 * 60 * 60   // max delay
+  );
+  await timelock.deployed();
+  return timelock;
+}
+
+async function makeBridgeReceiver({ initialize } = { initialize: true }) {
+  const [_defaultSigner, govTimelockAdmin, ...signers] = await ethers.getSigners();
+
+  const BaseBridgeReceiverFactory = (await ethers.getContractFactory('BaseBridgeReceiverHarness')) as BaseBridgeReceiverHarness__factory;
+  const baseBridgeReceiver = await BaseBridgeReceiverFactory.deploy();
+  await baseBridgeReceiver.deployed();
+
+  const govTimelock = await makeTimelock({ admin: govTimelockAdmin.address });
+  const localTimelock = await makeTimelock({ admin: baseBridgeReceiver.address });
+
+  if (initialize) {
+    await baseBridgeReceiver.initialize(
+      govTimelock.address,   // govTimelock
+      localTimelock.address  // localTimelock
+    );
+  }
+
+  return {
+    baseBridgeReceiver,
+    govTimelock,
+    localTimelock,
+    signers
+  };
+}
+
+function encodeBridgeReceiverCalldata({ targets, values, signatures, calldatas }) {
+  return utils.defaultAbiCoder.encode(TYPES, [targets, values, signatures, calldatas]);
+}
+
+describe('BaseBridgeReceiver', function () {
+  it('is initialized with empty storage values', async () => {
+    const { baseBridgeReceiver } = await makeBridgeReceiver({initialize: false});
+
+    expect(await baseBridgeReceiver.govTimelock()).to.eq(ethers.constants.AddressZero);
+    expect(await baseBridgeReceiver.localTimelock()).to.eq(ethers.constants.AddressZero);
+    expect(await baseBridgeReceiver.initialized()).to.eq(false);
+  });
+
+  it('initializing sets values', async () => {
+    const {
+      baseBridgeReceiver,
+      govTimelock,
+      localTimelock,
+    } = await makeBridgeReceiver({ initialize: false });
+
+    const tx = await wait(baseBridgeReceiver.initialize(govTimelock.address, localTimelock.address));
+
+    expect(await baseBridgeReceiver.govTimelock()).to.eq(govTimelock.address);
+    expect(await baseBridgeReceiver.localTimelock()).to.eq(localTimelock.address);
+    expect(await baseBridgeReceiver.initialized()).to.eq(true);
+
+    expect(event(tx, 0)).to.be.deep.equal({
+      Initialized: {
+        govTimelock: govTimelock.address,
+        localTimelock: localTimelock.address
+      }
+    });
+  });
+
+  it('cannot be reinitialized', async () => {
+    const {
+      baseBridgeReceiver,
+      govTimelock,
+      localTimelock
+    } = await makeBridgeReceiver({initialize: true});
+
+    await expect(
+      baseBridgeReceiver.initialize(govTimelock.address, localTimelock.address)
+    ).to.be.revertedWith("custom error 'AlreadyInitialized()'");
+  });
+
+  it('acceptLocalTimelockAdmin reverts if not called by localTimelock', async () => {
```

Comment:
nit: test name is inconsistent with the others, could be `acceptLocalTimelockAdmin > revert for unauthorized caller` as well

Reply:
updated